src/utils/stackingUtils.ts

```diff
@@ -173,6 +183,6 @@ export function stackedExtent(stackingResult: StackingResult, keyAccessor: IAcce
  * @param {any} key The key to be normalized
  * @return {string} The stringified key
  */
-export function normalizeKey(key: any) {
+export const normalizeKey = lMemoize((key: any) => {
```

Comment:
@themadcreator I _believe_ that any `key` passed in here should be immutable, but given it's typed as `any`, would want a sanity check?